aten/src/ATen/native/native_functions.yaml

```diff
@@ -2985,7 +2985,7 @@
 # to(Device) must not exist because all constructors of Device also works for
 # TensorOptions. Otherwise, an ambiguity error is thrown.
 # See NOTE [ TensorOptions Constructors ].
-- func: to(Tensor self, *, ScalarType dtype, Layout layout, Device device, bool non_blocking=False, bool copy=False) -> Tensor
+- func: to(Tensor self, *, ScalarType dtype, Layout layout, Device device, bool pin_memory, bool non_blocking=False, bool copy=False) -> Tensor
```

Comment:
This adds a new argument `pin_memory` without a default value and appears to be BC breaking. This is breaking some of our models. @VitalyFedyunin is it possible to add a default here?

Reply:
@jamesr66a do you have example of such breakage? Adding default just for one parameter here is pretty hard, but I can work on fixing your case. Having it as simple test would help a lot.

Reply:
See internal task